src/fastoad/notebooks/02_explanation/04_OpenMDAO.ipynb

```diff
@@ -0,0 +1,892 @@
+{
```

Comment:
<ul><li>"a <a href="https://openmdao.org/newdocs/versions/latest/features/core_features/working_with_groups/main.html" target="_blank">Group</a> component type" --&gt; "a <a href="https://openmdao.org/newdocs/versions/latest/features/core_features/working_with_groups/main.html" target="_blank">Group</a> class"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Corrected in a6b9f26f